src/index.js

```diff
@@ -8,7 +8,9 @@ import allReducers from './reducers';
 import { Provider } from 'react-redux';
 
 const store = createStore(allReducers, window.__REDUX_DEVTOOLS_EXTENSION__ && window.__REDUX_DEVTOOLS_EXTENSION__())
+fetch("https://randomuser.me/api/?results=10").then(response=>response.json()).then(response=>{
```

Comment:
Why fetch here?

Reply:
Redudant changes not related with issue #2